packages/verification-vue-component/src/style/components/verification/card/_base.scss

```diff
@@ -152,8 +152,8 @@
                 .verifier-image {
                     flex: 1 1 auto;
                     overflow: hidden;
-                    padding-top: 5px;
-                    height: 50px;
+                    margin-top: 5px;
+                    height: 25px;
```

Comment:
Does the switch redesign logo still look good with this new height?

Reply:
Yes and the smaller size is already approved by Steph and Fabian